.github/workflows/ci.yml

```diff
@@ -26,26 +42,103 @@ jobs:
 
       - uses: actions/cache@v3
         with:
```

Comment:
can these have names too?

Reply:
@leordev says he'll polish to add names; thanks!